mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,19 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a,b,c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+  name = "Sonjide";
+  age = 27;
+  return "Hello, my name is " + name + " and I am " + age + " years old";
+}
```

Comment:
the return string is correct but you dont want to reassign `name` and `age` to static values otherwise this function cant be used fore anyone that is not called Sonjide and happens to be 27 years old